src/main/java/com/libertymutualgroup/herman/aws/ecs/EcsPush.java

```diff
@@ -239,22 +232,18 @@ public EcsPush(EcsPushContext context) {
     public void push() {
         EcsPushDefinition definition = getEcsPushDefinition();
 
-        String accountId = this.stsClient.getCallerIdentity(new GetCallerIdentityRequest()).getAccount();
-        bambooPropertyHandler.addProperty("account.id", accountId);
```

Comment:
Any reason this was dropped in particular or did it get lost in a merge?

Reply:
"account.id" gets added as a prop value before this, see PropertyHandlerUtil.addStandardProperties. Both the task context prop handler and cli prop handler have this already.

Reply:
Ah, cool, I had lost track of where it was getting brought in, but the property handler is definitely the right spot.